namespaces/Representor.md

```diff
@@ -0,0 +1,47 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object)
```

Comment:
I would not copy what a member element does for a few reasons, but primarily because it is something that does not exist in the JSON world. Strings and numbers and such are easy, but when you have a key/value and you need an element to act as the glue, that element is kind of a rare one.

In reality, you should design your structures like you would with XML if you designing from the ground up. I have an idea to help this all, but we didn't get to it in our discussion. I'll put together some things on this soon.